src/core/resources/WindowControls.qml

```diff
@@ -0,0 +1,50 @@
+import QtQuick 1.1
+import DisplayCluster 1.0
+
+Rectangle {
+    id: controls
+    width: 300
+    height: 100
+    color: "#80000000"
+    border.color: "#a0000000"
+    border.width: 3
+    radius: 20
+
+    signal close()
+    signal toggleFullscreen()
+
+    Item {
+        width: parent.width * 0.95 / 2
+        height: parent.height * 0.95
+        anchors.left: parent.left
+        anchors.leftMargin: parent.radius
+        anchors.verticalCenter: parent.verticalCenter
+        Text {
+            text: "max/min"
+            color: "#a0000000"
+            font.pixelSize: 32
+            anchors.centerIn: parent
+        }
+        TouchArea {
```

Comment:
Does it then make sense to define the TouchArea type in the app and use it in the library? Move the resources to the app or the TouchArea to the lib I'd say, if there is no technical constraint